### PR TITLE
F/gcb 129 bonusly same bubble sizes

### PR DESCRIPTION
**Pull request by bartekchlebek**

### CODE REVIEW

**Comment by ochococo** [comment]
Please see this issue: https://github.com/macoscope/GrandCentralBoard/issues/30

**Comment by Augustyniak** [comment]
@bartekchlebek  Resolve merge conflicts

**Comment by bartekchlebek** [comment]
@Augustyniak 🎾

**Comment by Augustyniak** [comment]
Reviewing :eyes:

File: GrandCentralBoardTests/UpdateTests.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        
+        
+        let secondUpdate = updates[1]
+        expect(secondUpdate.bonus) == 50
+        expect(secondUpdate.name) == "bbb"
+        expect(secondUpdate.date) == NSDate(timeIntervalSince1970: 1457710241)
+        expect(secondUpdate.childBonuses.count) == 1
+        expect(secondUpdate.childBonuses[0].bonus) == 20
+        expect(secondUpdate.childBonuses[0].name) == "bbb"
+        expect(secondUpdate.childBonuses[0].date) == NSDate(timeIntervalSince1970: 1457711441)
```
Comment (Augustyniak) [inline GrandCentralBoardTests/UpdateTests.swift]: I don't find this line readable - can we somehow improve that?

File: GrandCentralBoardTests/UpdateTests.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        let jsonURL = NSBundle(forClass: self.dynamicType).URLForResource("UpdatesTests", withExtension: "json")
+        let json = NSData(contentsOfURL: jsonURL!)
+        
+        let updates = try! Update.updatesFromData(json!)
+        expect(updates.count) == 2
+        
+        let firstUpdate = updates[0]
+        expect(firstUpdate.bonus) == 10
+        expect(firstUpdate.name) == "aaa"
+        expect(firstUpdate.date) == NSDate(timeIntervalSince1970: 1457710841)
```
Comment (Augustyniak) [inline GrandCentralBoardTests/UpdateTests.swift]: I don't find this line readable - can we somehow improve that?

**Comment by Augustyniak** [comment]
Idea (for the next time): please add pods in a separate pull requests. Adding them together with other changes makes checking changes a pain.

**Comment by Augustyniak** [comment]
@bartekchlebek  I have no more comments.

**Comment by bartekchlebek** [comment]
@Augustyniak Should be better now. wdyt?

**Pull request merged**